client/grpc/node/service.go

```diff
@@ -37,6 +39,20 @@ func (s queryServer) Config(ctx context.Context, _ *ConfigRequest) (*ConfigRespo
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
 
 	return &ConfigResponse{
-		MinimumGasPrice: sdkCtx.MinGasPrices().String(),
+		MinimumGasPrice:   sdkCtx.MinGasPrices().String(),
+		PruningKeepRecent: s.cfg.PruningKeepRecent,
+		PruningInterval:   s.cfg.PruningInterval,
+	}, nil
+}
+
+func (s queryServer) Status(ctx context.Context, _ *StatusRequest) (*StatusResponse, error) {
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+
+	blockTime := sdkCtx.BlockTime()
+
+	return &StatusResponse{
+		// EarliestStoreHeight: sdkCtx.MultiStore(), // todo get earliest version from store
```

Comment:
nit: add a reference issue to the TODO
```suggestion
		// TODO: Get earliest version from store.
		//
		// Ref: ...
		// EarliestStoreHeight: sdkCtx.MultiStore(),
```